src/tests/test.shared.ts

```diff
@@ -4,10 +4,14 @@ import { sha256 } from "../utils";
 
 const request = supertest(app);
 
-export const sharedTests = (store: string, delayMs = 0) => {
+export const sharedTests = (store: string, purge: Function, delayMs = 0) => {
```

Comment:
You can retrieve purge's type with the following to avoid using the weaker `Function` type:

```
import expressAgressiveCache from "../";

type PurgeFunction = ReturnType<typeof expressAgressiveCache>["purge"];
```

Ideally we would export the purge function type as PurgeFunction directly from the index.ts similar to what's been done for the cache key function so that people consuming the library don't have to do the gymnastics I did above.

Reply:
I was hoping you would show me how to do this. My first time writing TypeScript. I will implement.